look more appropriate to have OI branding on the unlock screen.

--- mate-screensaver-1.26.1/data/org.mate.screensaver.gschema.xml.in.old	2022-08-04 16:50:17.899658532 +0000
+++ mate-screensaver-1.26.1/data/org.mate.screensaver.gschema.xml.in	2022-08-04 16:50:37.229786423 +0000
@@ -91,7 +91,7 @@
       <description>Allow the session status message to be displayed when the screen is locked.</description>
     </key>
     <key name="picture-filename" type="s">
-      <default>'@datadir@/backgrounds/mate/desktop/Stripes.png'</default>
+      <default>'/usr/share/pixmaps/backgrounds/openindiana/openindiana-default.jpg'</default>
       <summary>Picture Filename</summary>
       <description>File to use for the background image.</description>
     </key>
